recipes/acado/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.2.2":
```

Comment:
```suggestion
  "v1.2.2beta":
```

Reply:
Can conan package version contain letters?

Reply:
I tried it out locally, and I got the following line for a `find_package(ACADO 1.2.2 REQUIRED)`:
```
-- Found ACADO: 1.2.2beta (found suitable version "1.2.2beta", minimum required is "1.2.2")
```
I recently had problems with a lib that had a `v` prefix (like `v39`), and conan made `set(LIB_VERSION "39")` out of it, so CMake was not able to determine if the version is correct. [Also filed an issue](https://github.com/rollbear/trompeloeil/issues/222), and eventually ended up in [doing it properly](https://github.com/conan-io/conan-center-index/pull/3901) 😅 

TL&DR: Is it safe to deviate from strict semantic versioning?

Reply:
> TL&DR: Is it safe to deviate from strict semantic versioning?

100% CCI even has a costume virtual release notation `cci.yymmdd` for projects which do not have releases of are no longer maintained or outputting releases.

> According to SEMVER we should at least a add a dash, what do you think? 
EDIT: I tried it out, it works, so should we go for 1.2.2-beta ?

I personally think it's better to use the original upstream notation, it makes it easier to find what you want as a consumer.
However correcting the notation to be standard is also a nice thought

There's no rule (yet) so we'll see what the Conan team requests.

Perhaps a new topic for an official policy? /cc @jgsogo